boards/nxp/mimxrt1180_evk/mimxrt1180_evk.dtsi

```diff
@@ -81,3 +81,12 @@
 	pinctrl-0 = <&pinmux_lpi2c3>;
 	pinctrl-names = "default";
 };
+
+&flexcan3 {
+	status = "okay";
```

Comment:
enable this by default for M7 and M33 is not good

Reply:
Updated, set status in each core dts files.